.github/workflows/main.yml

```diff
@@ -30,6 +30,29 @@ jobs:
     with:
       run_only_integration: false
 
+  report_to_slack:
+    runs-on: ubuntu-latest
+    if: always() && github.ref == 'refs/heads/main'
```

Comment:
What is `always()` for? Not familiar with the semantics here, just trying to understand.